Fixed  a small bug with EES wrapper

arattner1 opened the pull request:
Added 'SI' directive to command string to make propssiz function work. Added a demo call to the EES sample file.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/CoolProp/CoolProp?pullRequest=1566) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/CoolProp/CoolProp?pullRequest=1566) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/CoolProp/CoolProp?pullRequest=1566) it.</sub>

ibell commented:
I can't see the diff and I can't use EES currently :(.  Can you describe the changes you made in text form please?

arattner1 commented:
Sure:
In the CoolProp.LIB EES library file, propssiz function, I just changed:

- `f$ = fluid$||'~'||output$||'~'||input1$||'~'||input2$`
to
- `f$ = fluid$||'~'||output$||'~'||input1$||'~'||input2$||'~'||'SI'`

This fixed the error message I was getting when calling that routine.

Also, I added an example of calling that function to the CoolProp_EES_Sample.EES. This solves implictly for the vapor/liquid concentrations in a two-phase two-component mixture.
`"Demonstration of using the PropSIZ function to solve for bubble & dew point concentrations at a  given temperature and pressure"
"Needs an initial guess for x_mol[1,1] to solve implicitly"

f$='n-Butane&n-Octane'
//x_mol[1,1] = 0.5
x_mol[1,2] = 1 - x_mol[1,1]
T        = 353.15[K]
P        = 90E3 [Pa]
P        = propssiz('P', 'T', T/1[K], 'Q', 0 , f$, 2, x_mol[1,1..2])*1[Pa]
//y_mol[1,1] = 0.5
y_mol[1,2] = 1 - y_mol[1,1]
P        = propssiz('P', 'T', T/1[K], 'Q', 1 , f$, 2, y_mol[1,1..2])*1[Pa]
`

ibell commented:
Thanks!
